README.md

```diff
@@ -208,9 +208,42 @@ $ xed .
 
 ## Environment variables
 
-`CATBIRD_MOCKS_DIR` — Directory where static mocks are located.
+- `CATBIRD_MOCKS_DIR` — Directory where static mocks are located.
+- `CATBIRD_RECORD_MODE` —  set this variable to `1` so that the application starts recording HTTP responses along the path set in `CATBIRD_MOCKS_DIR`. Default `0`.
+- `CATBIRD_REDIRECT_URL` — set this url to forward direct requests to catbird. By default, nil. If the recording mode is not enabled, then first the responses will be searched in the mocks and only if nothing is found, the request will be forwarded.
+- `CATBIRD_PROXY_ENABLED` — set this variable to `1` to forward proxy requests to catbird. By default, `0`. If the recording mode is not enabled, then first the responses will be searched in the mocks and only if nothing is found, the request will be proxied.
 
-`CATBIRD_PROXY_URL` — If you specify this URL Catbird will run in write mode. In this mode, requests to Catbird will be redirected to the `CATBIRD_PROXY_URL`. Upon receipt of response from the server it will be written to the `CATBIRD_MOCKS_DIR` directory.
+> Catbird supports proxying only HTTP requests. HTTPS requests are not supported!
```

Comment:
How do you think, it is possible in the future to make a proxy for HTTPS endpoints, like Charles or Proxyman?

Reply:
That would be cool! But Vapor does not support configuring the server as a proxy. To do this, you would have to use SwiftNIO without Vapor. I haven't been able to make a working prototype yet